examples/Suri.jl

```diff
@@ -526,7 +526,7 @@ function extend(A::RRSMat, P::Int)
 
   # have (f mod p)(a_i) - need f mod p  #TODO: list of p!
   fp = Dict{Int, Array{nmod_mat, 1}}()
-  k = 
+  k =
```

Comment:
:)

Reply:
😂 I guess this assigns the return value of the following loop, and hence does not raise an error? Of course it's still not very useful...